straxen/rucio.py

```diff
@@ -29,12 +37,10 @@ class RucioFrontend(strax.StorageFrontend):
                   'SDSC_USERDISK': r'.sdsc.'
                   }
     local_did_cache = None
-    local_rucio_path = None
-
-    # Some attributes to set if we have the remote backend
-    _did_client = None
-    _id_not_found_error = None
-    _rse_client = None
+    path = None
+    local_prefixes = {'UC_DALI_USERDISK': '/dali/lgrandi/rucio/',
+                      'SDSC_USERDISK': '/expanse/lustre/projects/chi135/shockley/rucio',
+                      }
```

Comment:
If this is for local use only could we put it in some local file then?

Reply:
Generally you are correct.

Reply:
Ok @WenzDaniel  I see your point. Those paths are quite secure I think but in principle I agree. I will look into other options, perhaps using utilix as @JoranAngevaare mentioned.